docs/src/mpolynomial.md

```diff
@@ -221,7 +221,10 @@ x^2*y^2+x+1//1
 Incase a specific parent ring is constructed, it can also be passed to the function.
 
 ```@docs
-change_base_ring(p::AbstractAlgebra.MPolyElem{T}, g, R::AbstractAlgebra.MPolyRing) where {T <: RingElement}
+change_base_ring(R::Ring, p::AbstractAlgebra.MPolyElem{T}, Rx::AbstractAlgebra.MPolyRing) where {T <: RingElement}
+change_base_ring(R::Ring, p::AbstractAlgebra.MPolyElem{T}) where {T <: RingElement}
+map(g, p::AbstractAlgebra.MPolyElem{T}, Rx::AbstractAlgebra.MPolyRing) where {T <: RingElement}
+map(g, p::AbstractAlgebra.MPolyElem{T}) where {T <: RingElement}
```

Comment:
I'm not convinced by calling this function `map` here. `map` is supposed to be applied to a "collection", whatever that means. Sure a polynomial can be seen as a collection of coefficients, but this seems a bit far stretched, and a polynomial doesn't currently behave as a collection (no iteration defined on it). I don't say it's necessarily a bad idea, but I think we should not rush into this new API. 
Also, the 3-arg version looks weird, usually `map` has two arguments, or if it has more, then all the arguments but the function `g` are meant to contain elements which will be passed as arguments to `g`.

Reply:
We can certainly define some iterators (coeffs and exponents) for polynomials. But it's certainly not a collection.

But on the other hand, I don't think there's anything that says that map should only be applied to collections. 

What else could we call this?

Reply:
Lets continue in #431

Reply:
This is #431. Did you mean something else?

Reply:
Oops, yes I meant #421 :)